packages/fether-react/src/Tokens/TokensList/TokensList.js

```diff
@@ -4,6 +4,8 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 import React, { Component } from 'react';
+
+import defaultTokenImage from '../../assets/img/tokens/default-token-128x128.jpg';
```

Comment:
You could import the image inside TokenBalance and NewTokenItem only, and directly pass it as prop to TokenCard. It'll save you a couple of props passing.

Reply:
totally, thanks